.github/workflows/main.yml

```
@@ -1,6 +1,13 @@
-on: [push, pull_request]
-env:
-  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+on:
+  push:
```

```suggestion
on:
  workflow_dispatch:
  push:
```

Enable a manual approach to triggering the GitHub Action.